source/2019-03-30-the-ember-times-issue-91.md

```diff
@@ -17,8 +17,11 @@ responsive: true
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Building a Hangman Game with Ember Octane](https://www.twitch.tv/videos/400723351) 🎬
```

Comment:
Sorry so much discussion on this one! The build is failing because of alex, our language checker, see: https://github.com/ember-learn/ember-blog/blob/master/source/CONTRIBUTING.md and https://travis-ci.org/ember-learn/ember-blog/builds/512207696#L1307-L1310. 

I think you'll need to add `<!--alex ignore hangman-hangwoman-->` above every line/paragraph with `hangman`. You can download alex and run it locally to see though...

Reply:
Added it here https://github.com/ember-learn/ember-blog/pull/30/commits/2f5a1a46bf19719073203e2989787dcddd79c685, checked locally, gonna merge to prevent merge conflicts / move forward on the main branch